From 8cfe7b6243660a5f1b43fe7217a678317284c430 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Tue, 14 Jun 2022 09:08:00 +0900
Subject: [PATCH] locale: rename keymap-util.[ch] -> localed-util.[ch]

As the file contains not only keymap related functions.

(cherry picked from commit 3e5203b3eb2b06ef346eef13d64430542bbfb5c4)

Related: #2087652
---
 src/locale/{keymap-util.c => localed-util.c}           |  3 +--
 src/locale/{keymap-util.h => localed-util.h}           |  0
 src/locale/localed.c                                   |  2 +-
 src/locale/meson.build                                 | 10 +++++-----
 src/locale/{test-keymap-util.c => test-localed-util.c} |  2 +-
 5 files changed, 8 insertions(+), 9 deletions(-)
 rename src/locale/{keymap-util.c => localed-util.c} (99%)
 rename src/locale/{keymap-util.h => localed-util.h} (100%)
 rename src/locale/{test-keymap-util.c => test-localed-util.c} (99%)

diff --git a/src/locale/keymap-util.c b/src/locale/localed-util.c
similarity index 99%
rename from src/locale/keymap-util.c
rename to src/locale/localed-util.c
index a3af396ebe..f4f8e5d168 100644
--- a/src/locale/keymap-util.c
+++ b/src/locale/localed-util.c
@@ -15,8 +15,7 @@
 #include "fileio.h"
 #include "fs-util.h"
 #include "kbd-util.h"
-#include "keymap-util.h"
-#include "locale-util.h"
+#include "localed-util.h"
 #include "macro.h"
 #include "mkdir-label.h"
 #include "nulstr-util.h"
diff --git a/src/locale/keymap-util.h b/src/locale/localed-util.h
similarity index 100%
rename from src/locale/keymap-util.h
rename to src/locale/localed-util.h
diff --git a/src/locale/localed.c b/src/locale/localed.c
index 9718c5b95f..2c324efb14 100644
--- a/src/locale/localed.c
+++ b/src/locale/localed.c
@@ -20,7 +20,7 @@
 #include "def.h"
 #include "dlfcn-util.h"
 #include "kbd-util.h"
-#include "keymap-util.h"
+#include "localed-util.h"
 #include "macro.h"
 #include "main-func.h"
 #include "missing_capability.h"
diff --git a/src/locale/meson.build b/src/locale/meson.build
index 0ccf71583d..2d8c48cf3a 100644
--- a/src/locale/meson.build
+++ b/src/locale/meson.build
@@ -1,9 +1,9 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 systemd_localed_sources = files('''
+        localed-util.c
+        localed-util.h
         localed.c
-        keymap-util.c
-        keymap-util.h
 '''.split())
 
 localectl_sources = files('localectl.c')
@@ -30,7 +30,7 @@ if conf.get('ENABLE_LOCALED') == 1
 endif
 
 tests += [
-        [files('test-keymap-util.c',
-               'keymap-util.c',
-               'keymap-util.h')],
+        [files('test-localed-util.c',
+               'localed-util.c',
+               'localed-util.h')],
 ]
diff --git a/src/locale/test-keymap-util.c b/src/locale/test-localed-util.c
similarity index 99%
rename from src/locale/test-keymap-util.c
rename to src/locale/test-localed-util.c
index f726e8e524..f658d271dc 100644
--- a/src/locale/test-keymap-util.c
+++ b/src/locale/test-localed-util.c
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
 #include "alloc-util.h"
-#include "keymap-util.h"
+#include "localed-util.h"
 #include "log.h"
 #include "string-util.h"
 #include "tests.h"
